src/MoneyFactory.php

```diff
@@ -54,7 +54,6 @@
  * @method static Money ATOM(numeric-string|int $amount)
  * @method static Money AUCTION(numeric-string|int $amount)
  * @method static Money AUD(numeric-string|int $amount)
- * @method static Money AUD(numeric-string|int $amount)
```

Comment:
How about removing fiat currencies in https://github.com/moneyphp/crypto-currencies?

I think it is more elegant to remove them at that stage because crypto-currencies library should provide only crypto currencies. The reason it provides some fiat currencies at the moment is that because they are listed listed on Binance. I can ignore them and provide only crypto currencies.

Reply:
Please do.

Reply:
I have created [an issue in the crypto currencies lib](https://github.com/moneyphp/crypto-currencies/issues/3). Do you time to fix this @slunak?

Reply:
Yeah, I will look after it. @frederikbosch I was looking into more options how to fetch crypto currencies from other available sources. Ideal scenario would be to fetch them from cryptomarketcap platform. The only caveat I see is that a person that fetches them will need to have a free accout with cryptomarketcap and use their API key. As I said it is ideal scenario because the platform has all the crypto currencies needed and does not include fiat currencies. What do you think about this scenario @frederikbosch ? If you are ok with it, then I will explore it more, otherwise I will just fix our current source, Binance. Downside of this approach is that in case Binance adds more fiat currencies then we will have to manually fix them too.

Reply:
I am not against that. I have not worked with secrets in Github actions, I guess there must an option for that.

Reply:
Ok, let me fix Binance first and then we call look into coinmarketcap option. P.S. I said cryptomarketcap in my previous comment, but I made a mistake, the platform is called coinmarketcap.